app/controllers/users_controller.rb

```diff
@@ -25,6 +25,21 @@ def show
     end
   end
 
+  def edit
+    @user = current_user
+  end
+
+  def update
+    current_user.update(name: params[:name],
+                        street_address: params[:street_address],
+                        city: params[:city],
+                        state: params[:state],
+                        zip_code: params[:zip_code],
+                        email: params[:email])
+    flash[:notice] = "Your profile has been updated"
+    redirect_to '/profile'
```

Comment:
Also maybe consider adding an exclamation! mark to a flash[:success] message, we want the user to feel excited that he/she updated their profile